pkg/securitycontext/accessors.go

```diff
@@ -191,6 +206,7 @@ type ContainerSecurityContextAccessor interface {
 	ProcMount() api.ProcMountType
 	SELinuxOptions() *api.SELinuxOptions
 	RunAsUser() *int64
+	RunAsGroup() *int64
```

Comment:
test in TestContainerSecurityContextAccessor and TestEffectiveContainerSecurityContextAccessor